### PR TITLE
Bug Fix in uploads - null checks for Wikidata Place

### LINKED ISSUE #4565: Upload does not progress, image not uploaded (linked to campaigns merge?)
Steps to reproduce:
1. Use latest master or any commit since 6588a6fd0ecf909111c198680
2. Tap + and the stock gallery button
3. Choose a picture, fill metadata, choose license
4. Push button to start upload
5. As usual, a grey image placeholder appears at the top of Contributions, with a progress bar below
6. Problem: No progress even after waiting for minutes

Commit 67f5b6c271135d4f19105e58e1665f83524601a4 does not have this problem, so it seems to be caused by campaigns code?

adb

### PR DESCRIPTION
**Pull request by ashishkumar468**
Fixes #4565

**Description (required)**

Fixes #4565 

What changes did you make and why?
Added null checks on contribution.getWikidataPlace()

**Tests performed (required)**

Tested beta Debug on API 19

### CODE REVIEW

**Comment by 4D17Y4** [comment]
Can u check by adding `if(FileUtils.fileExists(it) && FileUtils.fileExists(contribution.localUri))` before sha1 calculation. Would help me confirm if this is due to file deletion, thanks.

File: app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt [hunk 1]
```diff
@@ -290,7 +290,7 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
                         //Upload the file from stash
                             var countryCode: String? =null
                             with(contribution.wikidataPlace?.location){
-                                if(contribution.wikidataPlace?.isMonumentUpload!!) {
+                                if (contribution.wikidataPlace?.isMonumentUpload == true) {
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt:293]: Would `(contribution.wikidataPlace?.isMonumentUpload)` work too?

Comment (ashishkumar468) [reply app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt:293]: Hey Nicolas, contribution.wikidataPlace?.isMonumentUpload is Nullable. We would either have to use null assertion or an if else.

**Comment by nicolas-raoul** [comment]
> Can u check by adding `if(FileUtils.fileExists(it) && FileUtils.fileExists(contribution.localUri))` before sha1 calculation. Would help me confirm if this is due to file deletion, thanks.

And if it does not exist, logging a warning would be very helpful :-) Thanks a lot!

**Review by nicolas-raoul — APPROVED** [review]

**Pull request merged**